test/agent/Agent.js

```diff
@@ -0,0 +1,67 @@
+const agent = require('superagent');
+
+class Agent {
+  static async getRequest(url) {
+    const response = await agent.get(url)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    return response;
```

Comment:
every method is very similar and your goal is encapsule the authorization and headers, please check this library https://www.npmjs.com/package/superagent-defaults